audio-worklet/src/noisegate-audio-worklet.js

```diff
@@ -148,22 +166,24 @@ registerProcessor('noisegate-audio-worklet',
       releaseSteps = Math.ceil(this.sampleRate * this.release);
       releaseGainPerStep = 1 / releaseSteps;
     }
+    // For sine waves, the envelope eventually reaches an average power of
+    // a^2 / 2. Sine waves are therefore scaled back to the original
+    // amplitude, but other waveforms or constant sources can only be
+    // approximated.
+    const scaledEnvelopeValue;
+    const weight;
```

Comment:
How does this work?  These are const, but aren't initialized. And since they're const, they can't be changed.